README.md

```diff
@@ -93,7 +93,7 @@ This list of plugins will be merged with the list of built-in plugins, which wil
 
 ## Custom Tracing API
 
-The custom tracing API can be used to create custom trace spans. A *span* is a particular unit of work within a trace, such as an RPC request. Spans may be nested; the outermost span is called a *root span*, even if there are no nested child spans. Root spans typically correspond to incoming requests, while *child spans* typically correspond to outgoing requests, or other work that is triggered in response to incoming requests.
+The custom tracing API can be used to create custom trace spans. A *span* is a particular unit of work within a trace, such as an RPC request. Spans may be nested; the outermost span is called a *root span*, even if there are no nested child spans. Root spans typically correspond to incoming requests, while *child spans* typically correspond to outgoing requests, or other work that is triggered in response to incoming requests. This means that root spans shouldn't be created in a context where a root span already exists; a child span is more suitable here. Instead, root spans should be created to track work that happens outside of the request lifecycle entirely, such as periodically scheduled work.
```

Comment:
This is great. It would be nice to include and example of how to use the custom span API to create custom spans (perhaps in a new section after `Accessing the API` section below) along the lines of:

```ts
app.get('/', (req, res) => {
   const readFileSpan = tracer.createChildSpan({name: 'fs.readFile'});
   fs.readFile('/some/file', 'utf8', (err, data) => {
     readFileSpan.endSpan();
     res.send(data);
   });
});
```